docs/connections_and_transactions.md

```diff
@@ -95,6 +95,17 @@ async def create_users(request):
     ...
 ```
 
+When using with other decorators (such as route decorator `@router.post("")`
+from FastAPI), try to put `@database.transaction()` right above the function
+signature to ensure the automatic transaction management to be applied:
+
+```python
+@router.post("/my-awesome-endpoint")
+@database.transaction()
+async def awesome_endpoint():
```

Comment:
I think it would be better to mention something general that "If using multiple decorators, the `@database.transaction()` should be the most inner to the function definition. But maybe with better wording 😄